docroot/modules/custom/foia_personnel/src/Form/FoiaPersonnelSettingsForm.php

```diff
@@ -8,7 +8,7 @@
 /**
  * Class FoiaPersonnelSettingsForm.
  *
- * @ingroup foia_personnel
+ * FOIA Personnel.
  */
 class FoiaPersonnelSettingsForm extends FormBase {
 
```

Comment:
In this example you have removed `@ingroup foia_personnel`. I think that it is dangerous to remove anything that starts with `@` because they have special purposes outside of documentation. This is what I would recommend as a fix here:

```
/**
 * Render the personnel settings form.
 *
 * @ingroup foia_personnel
 */
```